File: subsystem_cpp/subsystem/user/rootviewcontroller.cpp
#include "rootviewcontroller.hh"
#include "cqlog.hh"

struct _self_RootViewController {
};

RootViewController::RootViewController() {
}

void RootViewController::viewDidLoad() {
    super::viewDidLoad();
    I("view load");
}

void RootViewController::viewDidAppear() {
    super::viewDidAppear();
    I("view appear");
}

void RootViewController::viewDidDisappear() {
    super::viewDidDisappear();
    I("view disappear");
}

File: subsystem_lua/subsystem/user/rootviewcontroller.lua
RootViewController = extends(CQViewController, {
})

function RootViewController:init()
    RootViewController.super.init(self)
end

function RootViewController:viewDidLoad()
    I("view load")
end

function RootViewController:viewDidAppear()
    I("view appear")
end

function RootViewController:viewDidDisappear()
    I("view disappear")
end
